src/runner.jl

```diff
@@ -16,7 +16,7 @@ function gen_runner_code(testfilename, logfilename, testreportsdir, test_args)
     display_reporting_testset(ts)
 
     write($(repr(logfilename)), report(ts))
-    exit(any_problems(ts))
+    any_problems(ts) && exit(2)
```

Comment:
Is there a particular reason to use `exit(2)` ?
Exit code 2 is reserved for  "Misuse of shell builtins"
https://tldp.org/LDP/abs/html/exitcodes.html

Reply:
No no reason, I incorrectly thought that 2 wasn't used. The const idea is good too, I'll create that and set it to 3